drivers/overlay/encryption.go

```diff
@@ -224,7 +225,9 @@ func programSA(localIP, remoteIP net.IP, spi *spi, k *key, dir int, add bool) (f
 	if add {
 		action = "Adding"
 		xfrmProgram = ns.NlHandle().XfrmStateAdd
-		crypt = &netlink.XfrmStateAlgo{Name: "cbc(aes)", Key: k.value}
+		salt := make([]byte, 4)
+		binary.LittleEndian.PutUint32(salt, k.tag)
```

Comment:
The key lamportClock is initialized from the cluster lamport clock, which in turns starts at 0 when the cluster is first created, and is therefore not close to random *at all*.

Can you ping santosh?